### ISSUE TITLE
Faster download with yt-dlp

### ISSUE THREAD

**Issue by pehmee** [ridoy/crate#8]
First things first; thanks for doing this plugin, i'm finding it very usable.

As you might know, yt-dl is very slow downloading from youtube nowadays. I downloaded yt-dpl, and change the filename to youtube-dl and now the Crate vst downloads instantly.

Maybe consider changing yt-dl to yt-dlp. Thanks for the plugin yet again <3

**Comment by ridoy** [comment]
Absolutely, I can switch it to yt-dlp when I get a chance. I'm very very glad you're finding the plugin usable, thank you for the feedback.
